### DISCUSSION TITLE
Usability: Rethink control elements appearing on mouseover

### DISCUSSION THREAD

**Discussion by wollmers** [RotherOSS/otobo#2269] [Ideas]
The use of a web-application should be as intuitive and self-explaining as possible.

Some experience from other applications can be assumed.

The disadvantages of elements only appearing at mouseover are:

- If not obvious a user must explore the page by moving the mouse across the area.
- The appearing element hides maybe some information.

The advantages are:

- Saves space in the layout.
- Provides information on demand, hides complexity (not in focus for the task).
- Saves clicks.

Candidates, where the behaviour should be removed:

- Configuration settings (disturbing, full menu exists)
- Ticket menu in ticket list view M (medium) and L (large).

Candidates, which would benefit from mouseover:

- Main menu for display of submenus, which many users would expect.
- Same for the avatar, which also has a submenu.
- Other menus with submenus e.g. Ticket Zoom.

**Comment by svenoe** [comment]
> Candidates, where the behaviour should be removed:
> 
>     * Configuration settings (disturbing, full menu exists)

What exactly do you mean?

>     * Ticket menu in ticket list view M (medium) and L (large).

Do you mean this?
<img width="815" alt="Bildschirmfoto 2021-01-04 um 14 52 35" src="https://user-images.githubusercontent.com/48322755/103542215-e8a62780-4e9c-11eb-826b-a60136ea2ddb.png">
I think it would be the main reason to use M or L over S, if people do this...

> Candidates, which would benefit from mouseover:
> 
>     * Main menu for display of submenus, which many users would expect.

This can be activated via "OpenMainMenu" in the SysConfig.

>     * Same for the avatar, which also has a submenu.

Agreed.

**Comment by wollmers** [comment]
> > Candidates, where the behaviour should be removed:
> > ```
> > * Configuration settings (disturbing, full menu exists)
> > ```
> 
> What exactly do you mean?
> 

This one:

![Bildschirmfoto 2021-01-04 um 15 32 56](https://user-images.githubusercontent.com/1275557/103548564-44c17980-4ea6-11eb-808c-89673552fcc8.png)

It has a conventional menu:

![Bildschirmfoto 2021-01-04 um 15 38 08](https://user-images.githubusercontent.com/1275557/103548934-d29d6480-4ea6-11eb-9d20-b3067d30b7bd.png)

**Comment by wollmers** [comment]
> > Candidates, which would benefit from mouseover:
> > ```
> > * Main menu for display of submenus, which many users would expect.
> > ```
> 
> This can be activated via "OpenMainMenu" in the SysConfig.
> 

Good to know;-)

I vote for enabling the setting `OpenMainMenu` by default.

**Comment by wollmers** [comment]
> > ```
> > * Ticket menu in ticket list view M (medium) and L (large).
> > ```
> 
> Do you mean this?
> <img alt="Bildschirmfoto 2021-01-04 um 14 52 35" width="815" src="https://user-images.githubusercontent.com/48322755/103542215-e8a62780-4e9c-11eb-826b-a60136ea2ddb.png">
> I think it would be the main reason to use M or L over S, if people do this...
> 

Exactly. M and L including the submenu makes sense. It's the need of mouse-over to display it, instead of always displaying the submenu or have a menu-icon in the upper right corner of each ticket entry.

**Comment by bschmalhofer** [comment]
Moved this from issues to discussion because the thread was more like a discussion.
Closing the discussion, as there surely will be new discussions about usabilitity.
